update-dependencies/Config.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Dotnet.Docker.Nightly
+{
+    public class Config
+    {
+        public static Config s_Instance { get; } = new Config();
+
+        private Lazy<string> _userName = new Lazy<string>(() => GetEnvironmentVariable("GITHUB_USER"));
+        private Lazy<string> _email = new Lazy<string>(() => GetEnvironmentVariable("GITHUB_EMAIL"));
+        private Lazy<string> _password = new Lazy<string>(() => GetEnvironmentVariable("GITHUB_PASSWORD"));
+        
+        private Lazy<string> _cliVersionUrl = new Lazy<string>(() => GetEnvironmentVariable("CLI_VERSION_URL", "https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/cli/rel/1.0.0"));
```

Comment:
Good point; the CLI version should always be explicitly passed.  What do you think about instead using a CLI_BRANCH environment variable, and then the CLI_VERSION_URL would be computed from it as follows:
`private Lazy<string> _cliVersionUrl = new Lazy<string>(() => string.Format("https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/cli/{0}", GetEnvironmentVariable("CLI_BRANCH")));`